<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manpal - Manual Palette Tuner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>151</width>
          <height>293</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="scrbars">
             <item>
              <widget class="QSlider" name="horizontalSlider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="pal_opts">
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>CheckBox</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="dit_ss">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Screen space dither</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Dither method:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item>
              <widget class="QLabel" name="somelabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="images" stretch="1,1">
           <item>
            <widget class="QLabel" name="srv_view">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>CrossCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="out_view">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello">
    <property name="title">
     <string>Test &amp;data</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuPalette">
    <property name="title">
     <string>Pa&amp;lette</string>
    </property>
    <addaction name="actionLoad_2"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionExport_PS"/>
    <addaction name="actionExport_PPM"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionSort_by_value"/>
    <addaction name="actionCreate_from_histogram"/>
    <addaction name="actionCreate_grayscale"/>
   </widget>
   <widget class="QMenu" name="menuWhatever_else">
    <property name="title">
     <string>&amp;mis&amp;c</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDefault_settings"/>
   </widget>
   <addaction name="menuHello"/>
   <addaction name="menuPalette"/>
   <addaction name="menuWhatever_else"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export GIMP</string>
   </property>
  </action>
  <action name="actionExport_PS">
   <property name="text">
    <string>Export &amp;PS</string>
   </property>
  </action>
  <action name="actionExport_PPM">
   <property name="text">
    <string>E&amp;xport PBM</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionReset_settings">
   <property name="text">
    <string>Reset settings</string>
   </property>
  </action>
  <action name="actionDefault_settings">
   <property name="text">
    <string>&amp;Reset settings</string>
   </property>
  </action>
  <action name="actionSort_by_value">
   <property name="text">
    <string>&amp;Sort</string>
   </property>
  </action>
  <action name="actionSort_by_hue">
   <property name="text">
    <string>Sort by hue</string>
   </property>
  </action>
  <action name="actionCreate_from_histogram">
   <property name="text">
    <string>&amp;Generate from histogram</string>
   </property>
  </action>
  <action name="actionAllocate">
   <property name="text">
    <string>How many colors</string>
   </property>
  </action>
  <action name="actionCreate_grayscale">
   <property name="text">
    <string>Ge&amp;nerate grayscale</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MainWin</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitter</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>MainWin</receiver>
   <slot>scaleSrc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitter_2</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>MainWin</receiver>
   <slot>scaleSrc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dit_ss</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWin</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWin</receiver>
   <slot>setDitherMethod(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>scaleSrc()</slot>
  <slot>preview()</slot>
  <slot>setDitherMethod(int)</slot>
 </slots>
</ui>
